modular_skyrat/modules/customization/modules/surgery/organs/pod.dm

```diff
@@ -0,0 +1,43 @@
+/obj/item/organ/external/pod_hair
+	name = "podperson hair"
+	desc = "Base for many-o-salads."
+
+	mutantpart_key = "pod_hair"
+	mutantpart_info = list(MUTANT_INDEX_NAME = "Ivy", MUTANT_INDEX_COLOR_LIST = list("#ffffff", "#ffffff", "#ffffff"))
+
+	zone = BODY_ZONE_HEAD
+	slot = ORGAN_SLOT_EXTERNAL_POD_HAIR
+
+	bodypart_overlay = /datum/bodypart_overlay/mutant/pod_hair
+
+/datum/bodypart_overlay/mutant/pod_hair
+	layers = EXTERNAL_FRONT_OVER|EXTERNAL_FRONT
+	color_swapped_layer = EXTERNAL_FRONT_OVER
+	color_source = ORGAN_COLOR_OVERRIDE
+
+/datum/bodypart_overlay/mutant/pod_hair/override_color(rgb_value)
+	return draw_color
+
+// TODO: at some point make this use tricolors instead of inverting
```

Comment:
Is that TODO still relevant?

Reply:
No, it's been implemented. There was some merge skew between my branch and this one because my branch didn't create a new file while this one did. The todo can be removed.